src/boardlib/__main__.py

```diff
@@ -64,14 +66,15 @@ def main():
     )
     args = parser.parse_args()
 
+    password = pyperclip.paste()
```

Comment:
I'd prefer not to add a dependency on `pyperclip` here. We could support env vars like `<board_name>_PASSWORD` and check to see if that exists before prompting for a password using `getpass`. E.g. `export KILTER_PASSWORD="myPassword" && boardlib kilter ...`. That would also let you use `pbpaste` or similar if you still wanted to do the pasting method, just pbpaste into the variable.

Reply:
env vars would be perfect! I was just having a hard time blindly typing in random text.